LDM-592.tex

```diff
@@ -171,15 +171,15 @@ \section{Alert Production}
 The user, a Staff Scientist, specifies a visit to be processed.
 
 \item
-The butler expands the incomplete data ID by setting the filter and selecting only those CCDs for which raw image data is available.
+The butler expands the incomplete DatasetExpresion by setting the filter and selecting only those CCDs for which raw image data is available.
```

Comment:
Not sure a `DatasetExpression` can be incomplete at all (apart from a list of `DatasetType`s), since it represents a SQL query.